### ISSUE TITLE
How do I get the tile information while I click on that tile

### ISSUE THREAD

**Issue by gauravshelangia** [moagrius/TileView#398]
Actually, I want to place the marker on the tile on which I clicked. So, for this I need the boundary of that tile. So please tell me how to do it.

**Comment by peterLaurence** [comment]
Do you really want to know the boundaries of the tile? Tell us more precisely what you want to do :

1. From a single tap place a marker in the center of the corresponding tile
2. From a single tap place a marker at the exact position of the touch event

This is possible, check out the FAQ [How do I get relative coordinates (e.g., latitude and longitude) from a pixel value like a touch event?](../wiki/FAQ) You don't need exactly what's given there, but you must understand that to implement your feature.

**It's easy to place a marker when you know the relative coordinates**

Check the Readme.md to understand what the relative coordinates are. If you want do the second scenario, getting those relative coordinates should be a good idea.
This idea is that a touch event is relative to your screen, while the TileView has its left and top borders scrolled when you pan the view. You can get how much the TileView is scrolled with the `TileView.getScrollX()` and `TileView.getScrollY()` methods. Beware that the values returned by those methods depend on the scale of the TileView. That's why convenience methods like `CoordinateTranslater().translateAndScaleAbsoluteToRelativeX()` exist. But you may not need that last method.

To sum up (the all-caps variable is in pixels):

ABSOLUTE_COORD_X = TileView.getScrollX() + touchEvent.getX()

Well, i can't explain everything now, as i would have to explain many things about how the TileView works in its core. At least you have clues to start thinking.

Post back if you are stuck.

**Comment by gauravshelangia** [comment]
I am new to android and I want to add marker on the tile on which I clicked. Please specify clearly, how to do this.

**Comment by moagrius** [comment]
thanks for trying @peterLaurence.  closing.

**Comment by gauravshelangia** [comment]
@peterLaurence  please tell me how to do the following: 
  From a single tap place a marker in the center of the corresponding tile
